packages/terra-base/src/Base.scss

```diff
@@ -4,16 +4,16 @@
 // More info: https://css-tricks.com/box-sizing/#article-header-id-6
```

Comment:
We can could flip this file to CSS, there is no reason it needs to be scss at this point.